AsyncDisplayKit/ASTextNode.mm

```diff
@@ -321,9 +353,14 @@ - (void)calculatedLayoutDidChange
   
   if (layout != nil) {
     ASDN::MutexLocker l(__instanceLock__);
-    if (CGSizeEqualToSize(_constrainedSize, layout.size) == NO) {
-      _constrainedSize = layout.size;
-      _renderer.constrainedSize = layout.size;
+    CGSize layoutSize = layout.size;
+    //textContainerInset
```

Comment:
Should we change the comment with a bit more information or just remove it. Comment seems kind of half done :)